# [001] Structure of the Human Genome

# Level of Genome Organisation

- Chromatin: double stranded helical structure of DNA.
- Nucleosome: DNA wrapped around histone proteins.
- Chromatosome: nucleosome and H1 histone.
- Chromosome: formed from 2 sister chromatids
- As you move from Chromatin to Chromosome, DNA is being progressively compressed further
- Chromosomes act as packaging units to allow for the compact storage of large amounts of DNA so that it can be stored in a small volume (E.g in the Nucleus) and is not tangled/damaged during the various stages of the cell cycle

![Screenshot 2022-02-25 at 17.23.53.png](%5B001%5D%20Structure%20of%20the%20Human%20Genome%2031c75f142fe54af2b69e36b011f10613/Screenshot_2022-02-25_at_17.23.53.png)

---

# Structure of the Genome

- Nuclear genome: 3 x 10‚Åπ base pairs and contains about 22,000 genes.
- Gene and gene-related sequences: 33% of nuclear genome.
- Coding DNA: around 22,000 genes ‚Üí Encodes proteins.
- Non-coding DNA: 90% ‚Üí This controls DNA expression.
1. Pseudogene
2. Gene fragments
3. Introns
- Extragenic DNA: 66% of nuclear genome.
- Unique or low copy number (Random DNA Sequences with little function regarding gene expression): 60%.
- Moderate to highly repetitive: 40%.
1. Tandemly repeated or clustered repeats: particular DNA sequence repeated in tandem/succession
2. Interspersed repeats: Repeat sequence is found in different locations all over genome. Various types of elements (LTR Segments found at the end of Retroviral Nuclei Acid Sequences, LINEs, SINEs, DNA Transposons that are capable of excising themselves out of the genome and reinsert themselves elsewhere in the Genome).

![Screenshot 2022-02-25 at 17.27.49.png](%5B001%5D%20Structure%20of%20the%20Human%20Genome%2031c75f142fe54af2b69e36b011f10613/Screenshot_2022-02-25_at_17.27.49.png)

---

# Typical Protein Encoding Eukaryotic Gene

- Upstream and Downstream enhancers can be found upstream (near the 5‚Äô end) or downstream (near the 3‚Äô end) from the gene which are involved in recruiting various general components that alter the expression of the gene
- At the 5‚Äô end of the Coding Strand (between Upstream enhancers and the TATA Box), Promoter Sequences can be found that are selective to particular transcription factors that are specific to and regulate the expression of their associated gene
- These are sometimes position independent and can be found at various regions either side of the gene but still influence the expression of their associated gene
- Between the Promoter Region and the 5‚Äô Untranslated Region (UTR), the TATA Box can be found, which indicates the coding strand and where transcription of that strand begins
- At the 5‚Äô and 3‚Äô End of the Gene, Non-Coding UTRs can be found, with the Coding Region between these UTRs being comprised of Exons and Introns
- Transcription occurs in a 5‚Äô to 3‚Äô direction and transcribes the 5‚Äô and 3‚Äô UTRs alongside the Exons and Introns in the Coding Region.
- Exons are coding segments of DNA that are transcribed and translated whereas Introns are non-coding segments of DNA that are found between exons and are transcribed but not translated (they are spliced out)
- Coding Region and UTRs are transcribed to form Immature (Still contains Introns) mRNA and following the addition of a 5‚Äô Cap and 3‚Äô Poly-A Tail (Increases mRNA stability), the Introns are spliced out of the mRNA to form mature mRNA that consists only of Exons and the UTRs and is released into the Cytoplasm so it can be translated

![Screenshot 2022-02-26 at 16.59.40.png](%5B001%5D%20Structure%20of%20the%20Human%20Genome%2031c75f142fe54af2b69e36b011f10613/Screenshot_2022-02-26_at_16.59.40.png)

---

# Exons and Introns

- Number of exons and introns in a gene is highly variable, but correlates to overall size of gene.
- Within a Gene, there are usually significantly fewer Exons than Introns
- Also, Exons generally are much smaller (fewer bps) than Introns
- HBB gene (haemoglobin ùõΩ): 3 exons and 2 introns.
- DMD gene (Duchenne muscular dystrophy): 79 exons.
- Usually, as Gene Size increases, the Number of Introns in the gene also increases, causing Exon:Intron Decreases
- E.g Histone H4 and Œ±-Interferon genes are very small and have no introns

---

# Genes within Genes

- Within Introns found in the coding region of Larger genes, Small genes can sometimes be found
- E.g Intron 26 of NF1/Neurofibromin 1 Gene contains 3 genes that are encoded in the region of the Antisense strand that is complementary to Intron 26 of the NF1 Gene (OGMP, EVI2A and EVI2B).

![Screenshot 2022-02-25 at 17.28.38.png](%5B001%5D%20Structure%20of%20the%20Human%20Genome%2031c75f142fe54af2b69e36b011f10613/Screenshot_2022-02-25_at_17.28.38.png)

---

# Gene Densities

- Across various regions of a chromosome, gene densities can vary so that some regions carry significantly more genes than other regions due to variations in the number of bp found within a single gene
- 6p21.3: on chromosome 6. In < 1 million bases, there are 70 genes.
- Xp21.2: on X chromosome. Dystrophin gene stretches over 2.4 million bases and contains numerous different exons and introns.

![Screenshot 2022-02-26 at 17.12.41.png](%5B001%5D%20Structure%20of%20the%20Human%20Genome%2031c75f142fe54af2b69e36b011f10613/Screenshot_2022-02-26_at_17.12.41.png)

---

# Gene Families

- Members may exhibit high sequence homology/DNA Sequence similarity due to a shared ancestry.
- These families produce genes that produce similar/related proteins with slightly different functions
- Sometimes contain a highly conserved domain (larger regions in the genes that are the same). E.g SOX box in SRY. Sequence found in SRY gene (Y chromosome) and is main gene expressed in male determination
- Sometimes contain a very short conserved DNA motif. E.g DEAD box (asp-glu-ala-asp) in RNA helicases.
- Superfamilies E.g Ig superfamily. These genes encode proteins that have different specific functions but a broadly related function
- Sometimes clustered in a similar region of a chromosome E.g Globin genes.
- Globin genes are found in 3 clusters (Myoglobin, Œ± Globin and Œ≤ Globin), with similar sequences suggesting a common ancestral origin, originally with a divergence of the Myoglobin from the Haemaglobins, followed by a division of the Haemaglobin Cluster into Œ± Globin and Œ≤ Globin clusters
- The Œ± Globin and Œ≤ Globin clusters containing many different types of proteins
- Œ± Globin Cluster is found on Chromosome 16 and Œ≤ Globin Cluster is found on Chromosome 11
- Œ± Globin = Œ∂ Globin (Embryonic), 3 Pseudogenes, Œ±1 Globin (Adult) and Œ±2 Globin (Foetal)
- Œ≤ Globin = Œµ Globin (Embryonic), Fetal Hb and Adult Hb with a single Pseudogene
- Different versions of Globin exist due to the vastly different oxygen environments (Different [Oxygen] and accessible forms of Oxygen as gaseous or aqueous) that the Embryo, Foetus and Adult are in and hence, the Hb in each of these developing stages must have varying oxygen affinities to maintain adequate oxygenation of each of these developmental stages despite vastly different oxygen environments

![Screenshot 2022-02-26 at 18.02.47.png](%5B001%5D%20Structure%20of%20the%20Human%20Genome%2031c75f142fe54af2b69e36b011f10613/Screenshot_2022-02-26_at_18.02.47.png)

- Often associated with truncated and non-processed pseudogenes within clusters. E.g salivary amylase.
- Pseudogenes = Genes that were once functional previously in evolution but have developed LoF Mutations that render them inactive

---

# Repetitive DNA Sequences

### Tandemly Repetitive DNA - 3 Types

| Class | Size of Repeated Sequence | Locations |
| --- | --- | --- |
| Satellite DNA (100 kb to several Mb) | 5 bp ‚Äì 5 kb | Heterochromatin particularly at centromeres. |
| Minisatellite DNA (0.1 ‚Äì 20 kb) | 6 ‚Äì 64 bp (Repeated more frequently in tandem than Microsatellites) | Scattered but concentrated
near telomeres. |
| Microsatellite DNA (< 100 bp) | 2 ‚Äì 5 bp | Dispersed throughout the genome. |
- Heterochromatin is variable in size, with variable numbers of Satellite DNA Repeats contributing to this variability
- Microsatellite Sequences have a relatively high probability of mutating to produce sequences with varying number of Microsatellite repeats (usually ¬±1 repeat)
- This produces a wide degree of variation of Microsatellite repeat length across the population, allowing them to be used in DNA Profiling, Paternity Testing and Population/Evolutionary Studies

### Ultracentrifugation of Satellite DNA

- DNA has a density in solution of 1.8g/ml (Heavier than Water which has a density of 1g/ml)
- When whole genomic DNA is centrifuged at a high speed in a high density solution of CsCl (similar chemically to NaCl), this causes a [CsCl] gradient moving up the tube where [CsCl] is greater at the bottom and lower at the top
- This causes the various components of DNA to separate according to density and be suspended in the region of the solution where the density of the solution is identical to that of the DNA Region.
- This forms DNA gradient bands within the solution where denser DNA regions are found at the bottom of the tube and less dense DNA regions are found at the top of the tube.
- Satellite bands are formed from Highly Repetitive DNA and are found above the main DNA band.
- As Satellite Sequences have a greater proportion of A-T than C-G Base Pairs within their highly repetitive DNA Sequences than Intergenic DNA, they are less dense than main DNA and hence, found higher up in the solution than the main Intergenic DNA Band).

![Screenshot 2022-02-25 at 17.33.48.png](%5B001%5D%20Structure%20of%20the%20Human%20Genome%2031c75f142fe54af2b69e36b011f10613/Screenshot_2022-02-25_at_17.33.48.png)

### Dispersed Repetitive DNA

| Class | Family | Number of Copies | % of Genome |
| --- | --- | --- | --- |
| SINE | Alu, MIR and MIR3 | 1,200,000, 450,000 and 85,000 respectively | 10.7%, 2.5% and 0.4% respectively |
| LINE - Longer than Alu Elements so LINE-1 Elements constitute a greater % of the Genome than Alu Elements | LINE-1,2 and 3  | 600,000, 370,000 and 44,000 respectively | 17.3%, 3.3% and 0.3% respectively |
| LTR Retroelements - Remnants of Retroviral cDNA in the Genome following infection | ERV and MaLR | 240,000 and 285,000 respectively | 4.7% and 3.8% respectively |
| DNA Transposons | MER-1 and 2  | 213,000 and 68,000 respectively | 1.4% and 1.0% respectively |
| Others |  | 60,000 | 0.4% |
- These are repeats scattered throughout genome, both within Intergenic and Extragenic Regions
- Alu Elements are typically found at the ends of Chromosomes (often in bands) but are found in varying numbers across different chromosomes
- Alu Repeats can also be found embedded within Genes but these elements almost always appear within non-coding introns because if found in the exon, they will cause a mutation that will alter the protein encoded by the gene.
- E.g Retinoblastoma 1 Gene contains 27 exons, with Alu, LINE-1 sequences interspersed, notably in the Introns

---

# Mitochondria

### Mitochondrial DNA (mtDNA)

- Following Endosymbiosis into Eukaryotic Cells, Mitochondria shed a significant proportion of their DNA but retained some
- However, the Mitochondrial Genome is not large enough to encode all of the Proteins found within the Mitochondria, so some mitochondrial proteins are also encoded by the Nuclear Genome to ensure that Mitochondria are dependent on the Nuclear Genome and hence, the Eukaryotic Cell for their function
- Unlike nuclear DNA, mtDNA is naked (not associated with histones or other proteins) and is circular.
- Each mitochondria also contains multiple copies of the mtDNA freely within the cytoplasm vs the Nuclear Genome where there is only 1 copy in each cell
- It is very Gene rich, containing little junk DNA that does not encode proteins (little extragenic and Non-Coding DNA):
1. 13 protein-coding genes (notably encoding Proteins directly responsible for Oxidative Phosphorylation).
2. 2rRNA genes.
3. 22 tRNA genes (so many because genetic code of mtDNA is slightly different to that used in nuclear genes and hence, requires different tRNA to the Nuclear Genome).
- tRNA genes are very small in size and hence, constitute a very small proportion of the Mitochondrial Genome
- The D Loop of the Mitochondrial Genome is non-coding but is involved in replication of the Mitochondrial Genome
- mtDNA has a ‚Üë mutation rate than nuclear DNA due to:
1. Action of free radicals and ROS generated during oxidative phosphorylation which oxidise and hence, damage the mtDNA
2. It is vulnerable to mutation as it is naked/not histone associated
3. It has limited DNA Repair Machinery and hence, a reduced capacity to correct mutations/damage
4. Enzyme responsible for Replication of Mitochondrial DNA has a higher error rate than Nuclear DNA Polymerase
- It is around 1/180,000th size of nuclear genome.

![Screenshot 2022-02-26 at 19.31.19.png](%5B001%5D%20Structure%20of%20the%20Human%20Genome%2031c75f142fe54af2b69e36b011f10613/Screenshot_2022-02-26_at_19.31.19.png)

### Mitochondrial DNA Code

- There are instances where the genetic code differs in mitochondria compared to nuclear DNA, meaning the ‚Äòuniversal‚Äô genetic code does not always apply to the Mitochondrial Genome.

| Codon | Nuclear Genes | Mitochondrial Genes |
| --- | --- | --- |
| UGA | Stop | Tryptophan |
| AGA | Arginine | Stop |
| AGG | Arginine | Stop |
| AUA | Isoleucine | Methionine |

### Mitochondrial DNA Inheritance

- Found in males and females.
- mtDNA is Inherited from mothers but not fathers as the sperm does not contribute its mitochondria to fertilised egg
- This is due to the small size of the Sperm in comparison to the Ovum and as Mitochondria in the Sperm are localised to the Neck region vs the Head (which contains the Nuclear Genome) and hence, when fertilisation occurs, the Nuclear Genome can enter the Ovum from the Head whereas the Mitochondria are unable to leave the Neck of the Sperm.
- Therefore, due to this inheritance pattern down the Maternal Line, it can be used to investigate female specific migration history/ancestry alongside the high mtDNA mutation rate meaning that mtDNA is highly variable within a population and unique to a specific family/individual.

![Screenshot 2022-02-26 at 19.40.31.png](%5B001%5D%20Structure%20of%20the%20Human%20Genome%2031c75f142fe54af2b69e36b011f10613/Screenshot_2022-02-26_at_19.40.31.png)

---

# Mitochondrial ‚ÄòEve‚Äô

- The whole mitochondrial genome (~16,569 bp) of 53 individuals was used to form a phylogenetic ancestral tree and identify the divergence point of the mtDNA within a population and hence, identify the common maternal ancestor (Mitochondrial Eve).
- TMRCA/Mitochondrial Eve = 171,500 years ago.
- All 3 earliest splits in the phylogenic tree separate African-only lineages from African and non-African lineages, with later branches separating Non-African heritages, indicating that the Human Species has its origin in Africa
- Youngest branch that contains both African and non-African
lineages dates to 52,000 years ago (+/- 27,500 years)

![Screenshot 2022-02-26 at 19.44.14.png](%5B001%5D%20Structure%20of%20the%20Human%20Genome%2031c75f142fe54af2b69e36b011f10613/Screenshot_2022-02-26_at_19.44.14.png)

---

# mtDNA from Neanderthals

- As human cells contain more copies of mtDNA (due to each cell having numerous mitochondria, each of which has many copies of the Mitochondrial Genome) than Nuclear DNA, mtDNA is found in a greater number in Archaeological Specimen than Nuclear DNA and hence, easier to identify and study/analyse
- Studies of mtDNA in Specimen from Neanderthals was used to show that Neanderthals are found on a separate branch of a Phylogenetic Tree than Humans (despite Humans containing some Nuclear DNA from Neanderthals, not inherited down the Maternal Line)

![Screenshot 2022-02-26 at 19.47.07.png](%5B001%5D%20Structure%20of%20the%20Human%20Genome%2031c75f142fe54af2b69e36b011f10613/Screenshot_2022-02-26_at_19.47.07.png)

---