# [008] Mutation

# Mutations

- Mutation: refers to any change in genetic information contained in an organisms‚Äô genes or chromosomes.
- First introduced by Dutch geneticist Huge De Vries in earl 1900s.

---

# Types of Chromosomal Mutation

### Translocation

- Section of chromosome is swapped with another chromosome so that genes usually found on one chromosome are now found on a different chromosome.
- If the breakage site for the translocation occurs part way along a gene, then the Translocated Chromosomes will contain a gene at the breakage site that consists of one part from the Original Chromosome and another from the Translocated Chromosome, producing a gene that encodes an abnormal protein or the normal gene product but it is expressed in the wrong condition (E.g if Promoter Region of Gene is translocted)
- Once chromosomes segregate, problems arise where chromosomes are formed from 2 segments from 2 different chromosomes.
- Many leukaemia‚Äôs are associated with translocations.

![Screenshot 2022-02-25 at 21.15.45.png](%5B008%5D%20Mutation%2038aaefb31082462db9477da2708a2869/Screenshot_2022-02-25_at_21.15.45.png)

### Insertion

- Section of chromosome inserted into another, leading to fusion gene formation at the breakage sites of the Chromosome after insertion, especially if the breakage site is found part way along a gene.
- Problems arise at segregation.

![Screenshot 2022-02-25 at 21.16.12.png](%5B008%5D%20Mutation%2038aaefb31082462db9477da2708a2869/Screenshot_2022-02-25_at_21.16.12.png)

### Deletion

- Removal of whole section of chromosome, tending to occur at ends of chromosomes.
- Results in abnormal gene dosing and can have significant effects if gene dosage of gene in deleted Chromosomal segment is important.
- Associated with Prader-Willi and Angelman syndrome.

![Screenshot 2022-02-25 at 21.17.02.png](%5B008%5D%20Mutation%2038aaefb31082462db9477da2708a2869/Screenshot_2022-02-25_at_21.17.02.png)

### Duplication

- Where a region of the Chromosome is repeated, usually in Tandem
- Results in abnormal gene loading onto a chromosome which can lead to dosage effects (overexpression of genes), which can produce clinical disease.
- It can alter the function of genes found at the duplication junction if this junction is found part way along a gene

![Screenshot 2022-02-28 at 08.34.33.png](%5B008%5D%20Mutation%2038aaefb31082462db9477da2708a2869/Screenshot_2022-02-28_at_08.34.33.png)

---

# DNA Mutations

- **Substitution** -  Most common type of mutation.
1. Transition -  Chemically like nucleotides are substituted for one another (G‚ÜíA and C‚ÜíT) (purine to purine or pyrimidine to pyrimidine).
- Transition more tolerable than transversion at single nucleotide level.
1. Transversion - All other changes (i.e. G‚ÜíT or G‚ÜíC) (purine to pyrimidine or pyrimidine to purine).
- There are 2x possible types of Transversion Mutation than Transition Mutation yet transition mutations are more common than Transversion Mutations
- This may be involved in the Evolution of the Genetic Code as Codons are tolerant to mutations (still encode the same AA or Start/Stop Codon) at the 3rd Position, producing redundancy in the Genetic Code
- For some codons where the 3rd AA encode either an AA or a Stop Codon/different AA, Transition Mutations (which are more common) do not alter what is encoded by the Codon whereas as Transversion Mutation alters what is encoded by the Codon
- **Deletion**
1. Random Deletion
2. Polymerase Slippage
3. Uneven crossing over
- **Addition**
1. Viral insertion - Retroviral insertion (notably of LTRs), where Retroviral cDNA is inserted into host genome and persists.
- These LTRs can insert into an exon where they cause LoF mutations or LTRs can often be strong enhancers and hence, they can upregulate gene expression, notably in new tissues where they previously were not expressed
1. Polymerase slippage
2. Uneven crossing over
- **Transposition**
- Transposons -  Region of DNA excised and re-inserted somewhere else in genome.

![Screenshot 2022-02-28 at 08.46.32.png](%5B008%5D%20Mutation%2038aaefb31082462db9477da2708a2869/Screenshot_2022-02-28_at_08.46.32.png)

| Mutagen | Effect | Outcome |
| --- | --- | --- |
| Formation of rare tautomeric form of bases via alterations of its stereochemistry (without breaking/gaining bonds). | Tautomeric forms of bases pair with different bases than their normal forms would, causing the Tautomeric Base to functionally act as another type of base  | G ‚Üí A, A ‚Üí G, T ‚Üí C, C ‚Üí T |
| Alkylating agents (e.g.
ethyl methane
sulfonate (EMS), nitrosoguanidine (NG)). | Introduce alkyl groups into nucleotides at
numerous positions, altering the base so that it pairs with a different base than normal (acts functionally as a different base).
Can also trigger misrepair of DNA. | G ‚Üí A, T ‚Üí C |
| Deamination (nitrous acid/HNO‚ÇÇ) | Deaminate nucleotides (NH‚ÇÇ groups replaced with O groups) to form a new base.
Cytosine ‚Üí Uracil
Adenine ‚Üí
Hypoxanthine | C ‚Üí T (Uracil which is a RNA Thymine Analogue), A ‚Üí G |
| Deamination (heat) | Causes deamination of
cytosine to Uracil in a similar way to nitrous acid. Also converts guanine
to cytosine
(purine-pyrimidine transversion) | C ‚Üí T, G ‚Üí C
(transversion) |
| UV light | Causes dimerization of
adjacent pyrimidines,
particularly thymine residues. These distort
structure of DNA and
prevent normal base pairing. | Requires repair
for normal
replication via removal of dimerised pyrimidines and replacement with non-dimerised pyrimidines |
| Ionising radiation (e.g.
X rays, ùú∏ rays) | Causes formation of
intracellular free radicals: molecules
containing unpaired
electrons. Cause damage to individual
bases and single or
double-stranded breaks (often causes Apoptosis to prevent translocations or other mutations). Broken DNA ends can join with breaks on other
chromosomes | Various
chromosomal
rearrangements where fragments of DNA reattach to other chromosomes. |
| Hydroxylamine
(NH‚ÇÇOH) | Reacts with cytosine
to produce N‚Å¥-Hydroxycytosine, which pairs Adenine/acts functionally as Thymine | C ‚Üí T |
| Viral insertion | Many viruses,
especially retroviruses,
incorporate their genome into host
DNA. This can knock out genes or alter
their expression if the retrovirus inserts near the gene as Retroviruses can also act as powerful
enhancers. | Various |
| Bulky adducts (e.g.
aflatoxin B) | Large chemical groups that form covalent
associations with DNA, particularly bases. These bases are then
removed to leave apurinic sites where no bases are found. | Various |
| Intercalators (e.g. Ethidium bromide, acridine orange). | Flat molecules that fit into a groove of DNA or in the stacked bases found along the DNA Molecule. Can interfere with DNA replication and
repair, and cause slipped-strand
mispairing. | Insertions and
deletions |

![Tautomeric Forms](%5B008%5D%20Mutation%2038aaefb31082462db9477da2708a2869/Screenshot_2022-02-28_at_08.52.43.png)

Tautomeric Forms

![UV Light](%5B008%5D%20Mutation%2038aaefb31082462db9477da2708a2869/Screenshot_2022-02-28_at_08.54.19.png)

UV Light

![Alkylkating Agents](%5B008%5D%20Mutation%2038aaefb31082462db9477da2708a2869/Screenshot_2022-02-28_at_08.53.35.png)

Alkylkating Agents

![Deamination](%5B008%5D%20Mutation%2038aaefb31082462db9477da2708a2869/Screenshot_2022-02-28_at_08.53.55.png)

Deamination

![Intercalators](%5B008%5D%20Mutation%2038aaefb31082462db9477da2708a2869/Screenshot_2022-02-28_at_09.12.57.png)

Intercalators

### Aflatoxin B

- Thought to be 1 of major mutagens found in food in 3rd world, particularly in tropics.
- Produced by fungus Aspergillus flavus oryzae, which grows on grain, peanuts etc. when stored in damp conditions.
- Converted to its reactive form, aflatoxin B1-8,9-exoepoxide, by cytochrome p450 in liver
- Intercalates and binds to guanine.
- Results in the removal of guanine residue in order to remove Aflatoxin B to form an apurinic site that is often filled by non-guanine bases, producing a substitution mutation.
- Strongly associated with hepatocellular carcinoma.

### Point Mutation

- Silent mutation (3rd base pair): change in 3rd base pair, but amino acid remains same due to redundancy of the genetic code.
- Usually this has no effect but sometimes, codon preference (where some codons more efficiently code for a particular AA than other codons which encode the same AA due to these codons having a higher complementarity to the AA-tRNA) means a silent mutation may alter the likelihood that an AA is encoded for by a codon
- Missense: change in amino acid coded for.
- Nonsense: change of amino acid to premature stop codon, shortening the protein.
- Frameshift mutation: losing or gaining bases (non-multiple of 3 - Addition/Deletion of 3z bases adds/removes an AA) moving reading frame, altering the AA encoded by codons downstream to the mutation but often cause a premature stop codon to arise in the new reading frame, truncating the protein.

### Splice Site Mutation

- Splice Donor Sites are found at the beginning/5‚Äô end of an Intron and Splice Acceptor Sites are found at the end/3‚Äô end of an Intron
- Splice donor mutation: does not splice out introns due to mutation (Mutated site does not indicate the start of the intron to indicate its position within the immature mRNA)
- Splice acceptor mutation: splices out exon alongside the two introns either side (as mutated site does not indicate the end of the First Intron so it is spliced out with the downstream exon and intron, where the next splice acceptor site is recognised on immature mRNA).

![Screenshot 2022-02-25 at 21.34.41.png](%5B008%5D%20Mutation%2038aaefb31082462db9477da2708a2869/Screenshot_2022-02-25_at_21.34.41.png)

---

# Mechanisms of Mutation

### Mechanism for Short Tandem Repeat Number Variation - DNA Polymerase Slippage (Slipped Strand Mispairing)

- An important source of variation in repeat regions.
- It can occur in homopolymer tracts (e.g. ‚ÄòCCCCCCC‚Äô) but in other short tandemly repeated DNA Sequences (E.g. Microsatellites), where the leading/coding strand temporarily dissociates and reassociates with the template strand more upstream than normally (often base adjacent to the site of dissociation for homopolymer tracts), causing the section of DNA between the dissociation and reassociation sites to form a hairpin loop/loop out and not associate with the template strand, causing mismatch between the DNA Sequence.
- This looped out region of DNA persists but DNA Repair Machinery extends the homopolymer sequence of the Coding Strand to replace the looped out bases so that the repeat length on the coding strand is now the same (not shorter) than the Template Strand/correct the mismatch
- When the hairpin loop is reinserted into the DNA in the next cycle of replication, the template strand is also extended so that the homopolymer repeat length on both strands is identical, causing a net increase in the repeat size equal to the number of bases that were in the Hairpin loop
- Most probable cause of repeat-number variation in microsatellites.

![Screenshot 2022-02-25 at 21.35.34.png](%5B008%5D%20Mutation%2038aaefb31082462db9477da2708a2869/Screenshot_2022-02-25_at_21.35.34.png)

### Fragile X Syndrome

- In Fragile X Syndrome, Increase in Repeat Length of Trinucleotide Microsatellite Sequences in the 5‚Äô UTR due to Slipped Strand Mismatch over successive generations can mutate the associated gene to cause Fragile X Syndrome

![Screenshot 2022-02-28 at 09.29.52.png](%5B008%5D%20Mutation%2038aaefb31082462db9477da2708a2869/Screenshot_2022-02-28_at_09.29.52.png)

### Mechanisms for Insertions/Deletions - Unequal Crossing Over

- Incorrect alignment of homologous chromosomes.
- As Recombination occurs at regions along the Chromosome where high homology is found (sequences are alike and hence, likely to be the homologous chromosomal segments with the same genes), there is the possibility of a Chiasmata forming between 2 different genes (in different chromosomal loci) that have a high similar homology (E.g Related Genes or at repeat sequences)
- Unequal Crossing over results in a large insertion in 1 molecule and a large deletion in another molecule (often on a Chromosomal level) as the lack of gene homology at the Chiasmata means the 2 chromosomes produced are no longer homologous.
- Can also cause expanded trinucleotide repeats where the Chiasmata occurs part way along a Microsatellite Repeat Sequence.

![Screenshot 2022-02-28 at 09.32.48.png](%5B008%5D%20Mutation%2038aaefb31082462db9477da2708a2869/Screenshot_2022-02-28_at_09.32.48.png)

### Mechanism for Recurrent Deletion - Misalignment and Recombination

- IIf there is a gene which has a similar repeat sequence on either side of it, DNA Recombination Machinery can lead to the 5‚Äô Repeat lining up with the 3‚Äô Repeat on the same Chromosome, causing the gene between these regions to loop out, causing large deletions of genes found in the Hairpin Loop

![Screenshot 2022-02-25 at 21.39.45.png](%5B008%5D%20Mutation%2038aaefb31082462db9477da2708a2869/Screenshot_2022-02-25_at_21.39.45.png)

---

# Mutation Frequency

- Whilst mutation rates per base pair are relatively rare, the sheer number of bases found within the human genome means that each cell acquires an appreciable amount of mutations with each replication cycle
- Point Mutations occur more frequently in males than females as male germ cells undergo more rounds of replication/proliferation than female germ cells

| Point Mutation Mechanism | Frequency/Cell Division |
| --- | --- |
| Mistakes in DNA replication. | 10‚Åª¬π‚Å∞/Basepair or 10‚Åª‚Åµ/gene or 0.5/cell |
| DNA damage by chemical mutagens (or by radiation) and misrepair. | 10‚Åª¬π‚Å∞/Basepair or 10‚Åª‚Åµ/gene or 0.5/cell |

| Submicroscopic Deletion or Insertion Mechanism | Frequency/Cell Division |
| --- | --- |
| Unequal crossing over | 10‚Åª¬π‚Å∞/Basepair or 10‚Åª‚Åµ/gene or 0.5/cell |
| Misalignment during DNA replication (Similar to Slipped Strand Mispairing). | 10‚Åª¬π‚Å∞/Basepair or 10‚Åª‚Åµ/gene or 0.5/cell |
| Insertion of mobile element (E.g Transposons and LTRs from Retroviruses). | 10‚Åª¬π‚Å∞/Basepair or 10‚Åª‚Åµ/gene or 0.5/cell |
| DNA damage by chemical mutagens (or by radiation) and misrepair | 10‚Åª¬π‚Å∞/Basepair or 10‚Åª‚Åµ/gene or 0.5/cell |

| Microscopically Visible Deletion, Translocation or Inversion Mutation Mechanism | Frequency/Cell Division |
| --- | --- |
| Unequal crossing over. | 6 x 10‚Åª‚Å¥ |
| DNA damage by chemical mutagens (or by radiation) and misrepair (would likely cause Apoptosis). | 6 x 10‚Åª‚Å¥ |

| Loss of Whole Chromosome Mutation Mechanism | Frequency/Cell Division |
| --- | --- |
| Missegregation at mitosis. | 1 in 100 |

---

# Mosaicism

- A mutation occurring after fertilisation, but relatively early (pre-embryo stage), then daughter cells will have mutation.
- Cellular fate of these mutated and non-mutated cells is not yet decided so as these cells proliferate, they form a mosaic of mutated and non-mutated cells within the Embryo, causing mutated cells to be present in various different types of tissues (alongside non-mutated cells) to produce a mosaic of mutated and non-mutated cells in various tissues in the body.
- Mutations occurring later tend to be more localised (E.g to one tissue or a few related tissues) as the mutated cells here are more differentiated and hence, less likely to differentiate to form a wide range of tissues in the body.

![Screenshot 2022-02-25 at 21.49.29.png](%5B008%5D%20Mutation%2038aaefb31082462db9477da2708a2869/Screenshot_2022-02-25_at_21.49.29.png)

---

# Dominant Mutants

- Usual expected result of a mutation is a loss of function.
- Thus, it is generally expected that mutant alleles would be recessive and therefore, not alter the phenotype of the individual (be silent) if the allele on the Homologous chromosome is normal.
- This explains why Autosomal Recessive is the most common inheritance pattern seen in the Population regarding genetic diseases

---

# Dominance

### Haploinsufficiency (Having 1 Copy of a Functional Gene is not enough to maintain normal function)

- Example: Hereditary haemorrhagic telangiectasia.
- Disease is caused by mutations in Endoglin gene (chromosome 9). Codes for endoglin, a growth hormone receptor which is expressed in endothelial cells lining blood vessels (TGF-ùõΩ receptor).
- As Endoglin Gene expression is proportional with the number of Endoglin molecules expressed, Heterozygous or Homozygous Recessive individuals express insufficient amounts of Endoglin which leads to disease (Both genes must be functional and expressed to establish normal phenotype).
- Causes multiple arteriovenous malformations (AVMs) that lack intervening capillaries and result in direct connections between arteries and veins.

### Dominant Negative Effect (Making a Not Quite Right Version of a Protein Interferes with Correct Form of Protein)

- Product of mutant interferes with action of normal allele.
- This is usually because protein needs to form a multimer to be active.
- 1 defective component inserted into multimer can destroy activity of whole complex as all components need to be normal for the entire protein complex to be functional.
- Example: osteogenesis imperfecta due to mutated collagen type I gene.
- Collagen fibrils are built of arrays of triple-helical procollagen units.
- Type I procollagen comprises 2 chains encoded by COL1A1 gene and 1 encoded by COL1A2.
- Mutations to COL1A1 or COL1A2 produce mutant chains that disrupt the structure and hence, function of the whole collagen fibril.
- Null-mutants (mutation causes no expression) are less severe as they produce less disruption of the protein complex

![Screenshot 2022-02-28 at 10.02.05.png](%5B008%5D%20Mutation%2038aaefb31082462db9477da2708a2869/Screenshot_2022-02-28_at_10.02.05.png)

### Gain of Function

- Most mutations either have no effect or cause loss of function, but occasionally, mutation can cause gain of an undesirable function.
- Example: ùú∂-1-antitrypsin. Normal role is to inhibit elastase in lung. When a mutation rends this gene inactive (LoF), there is excessive elastin breakdown in the Lung and Liver, causing Bronchial Hypersensitivity
- A rare single nucleotide change (Pi Pittsburgh) can change its specificity from elastase to thrombin, causing it to inactive Thrombin.
- This results in serum thrombin being degraded leading to a form of haemophilia.
- Therefore, due to its effects on Thrombin Depletion in Heterozygous or Homozygous individuals, it is dominant

### Dominance at the Level of the Organism but Recessivity at the Level of the Cell

- Example: tumour suppressor genes (retinoblastoma).
- 2 mutant alleles in a cell will result in tumour growth via unregulated growth.
- But, if an individual inherits only 1 mutant allele, then most cells will be protected against unregulated growth by the normal allele (recessive mutation).
- However, a small number of cells in the body will acquire an additional somatic mutations in normal allele which rends it dysfunctional, leading to a tumour. Thus, an individual inheriting only 1 mutant allele who acquires a somatic mutation in the healthy allele will develop tumours (dominant).

---